### PR TITLE
Coding standard: Use wearerequired/coding-standards

### PR DESCRIPTION
**Pull request by ocean90**
Updates PHPCS config to use https://github.com/wearerequired/coding-standards and fixes all new issues.

### CODE REVIEW

File: phpcs.xml.dist [hunk 1]
```diff
@@ -1,36 +1,34 @@
 <?xml version="1.0"?>
-<ruleset name="WordPress Coding Standards for Plugins">
-	<description>Generally-applicable sniffs for WordPress plugins</description>
+<ruleset name="rest-likes">
+	<description>Coding standards for WordPress plugins</description>
```
Comment (grappler) [inline phpcs.xml.dist]:
```suggestion
	<description>Coding standards for REST Likes</description>
```

File: classes/Comments.php [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
 /**
  * REST Likes Controller for comments.
- *
- * @package rest-likes
  */
 
 namespace Required\RestLikes;
 
 use WP_Comment_Query;
-use WP_Error;
```
Comment (grappler) [inline classes/Comments.php]: Wieso würde `WP_Error` entfernt aber `WP_Comment_Query` nicht?

Comment (ocean90) [reply classes/Comments.php]: Siehe https://github.com/wearerequired/rest-likes/pull/120/commits/b9fe203255d8804435866915388ee3297793b2e2. Kommt von der Regel `SlevomatCodingStandard.Namespaces.FullyQualifiedExceptions`.

`WP_Comment_Query` wird weiterhin für [Type Hinting](https://github.com/wearerequired/rest-likes/blob/efc66135bbf0a8f3df2459ef6f5f9f6a3d362dea/classes/Comments.php#L197) verwendet.

**Comment by swissspidy** [comment]
# [Codecov](https://codecov.io/gh/wearerequired/rest-likes/pull/120?src=pr&el=h1) Report
> Merging [#120](https://codecov.io/gh/wearerequired/rest-likes/pull/120?src=pr&el=desc) into [master](https://codecov.io/gh/wearerequired/rest-likes/commit/65bd2efb4dfcaa5bd63874d985811ef70df97dbb&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/wearerequired/rest-likes/pull/120?src=pr&el=tree)

```diff
@@           Coverage Diff            @@
##             master    #120   +/-   ##
========================================
  Coverage      0.00%   0.00%           
+ Complexity       98      95    -3     
========================================
  Files             4       4           
  Lines           304     304           
========================================
  Misses          304     304           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/wearerequired/rest-likes/pull/120?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/wearerequired/rest-likes/pull/120?src=pr&el=footer). Last update [65bd2ef...8d77265](https://codecov.io/gh/wearerequired/rest-likes/pull/120?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Pull request merged**